App/Sources/Feature/LoginFeature/Sources/LoginView.swift

```diff
@@ -8,72 +9,91 @@ struct LoginView: View {
-                    text: $viewModel.password,
-                    helpMessage: viewModel.passwordHelpMessage,
-                    link: "비밀번호 찾기",
-                    isError: viewModel.isPasswordErrorOcuured
-                )
-                .textContentType(.password)
-                .focused($focusField, equals: .password)
-            }
-            .padding(.top, 80)
-            .padding(.horizontal, 28)
-
-            Spacer()
-
-            VStack(spacing: 0) {
-                BitgouelButton(
-                    text: "로그인",
-                    action: {
-                        viewModel.login()
+                VStack {
+                    BitgouelTextField(
+                        "이메일",
+                        text: $viewModel.email,
+                        helpMessage: viewModel.emailHelpMessage,
+                        isError: viewModel.isEmailErrorOccured
+                    ) {
+                        focusField = .password
                     }
-                )
-                .cornerRadius(8)
-                .disabled(viewModel.isFormEmpty)
+                    .textContentType(.emailAddress)
+                    .focused($focusField, equals: .email)
+
+                    SecureBitgouelTextField(
+                        "비밀번호",
+                        text: $viewModel.password,
+                        helpMessage: viewModel.passwordHelpMessage,
+                        link: "비밀번호 찾기",
+                        isError: viewModel.isPasswordErrorOcuured
+                    )
+                    .textContentType(.password)
+                    .focused($focusField, equals: .password)
+                }
+                .padding(.top, 80)
                 .padding(.horizontal, 28)
 
-                Text("또는")
-                    .bitgouelFont(.caption, color: .greyscale(.g7))
-                    .padding(.top, 8)
+                Spacer()
+
+                VStack(spacing: 0) {
+                    BitgouelButton(
+                        text: "로그인",
+                        action: {
+                            viewModel.login()
+                        }
+                    )
+                    .cornerRadius(8)
+                    .disabled(viewModel.isFormEmpty)
+                    .padding(.horizontal, 28)
+
+                    Text("또는")
+                        .bitgouelFont(.caption, color: .greyscale(.g7))
+                        .padding(.top, 8)
 
-                Text("회원가입")
-                    .bitgouelFont(.text3, color: .primary(.p5))
-                    .padding(.top, 2)
+                    Button(
+                        action: {
+                            viewModel.signupPageIsRequired()
+                        },
+                        label: {
+                            Text("회원가입")
+                                .bitgouelFont(.text3, color: .primary(.p5))
+                                .padding(.top, 2)
+                        }
+                    )
```

Comment:
```suggestion
                    Button {
                        viewModel.signupPageIsRequired()
                    } label: {
                            Text("회원가입")
                                .bitgouelFont(.text3, color: .primary(.p5))
                                .padding(.top, 2)
                    }
```
trailing 클로져 적극 활용 어떤가요

Reply:
오!!! 좋아요!! 더 보기 좋네요